mmv1/templates/terraform/iam/storage_managed_folder_diff_suppress.go.erb

```diff
@@ -0,0 +1,14 @@
+func <%= object.resource_name -%>DiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
If this would be preferable please let me know and I'll implement it. In the meantime I've added https://github.com/hashicorp/terraform-provider-google/issues/18415 to track implementation of this feature.

Reply:
Any Updates on this?

Reply:
With the handwritten IAM resources this is not necessary anymore.